src/boot/efi/boot.c

```diff
@@ -87,6 +91,8 @@ typedef struct {
         BOOLEAN auto_entries;
         BOOLEAN auto_firmware;
         BOOLEAN reboot_for_bitlocker;
+        BOOLEAN auto_enroll_keys;
+        BOOLEAN allow_unsafe_enrollement;
```

Comment:
conceptually these aren#t two independent bools, are they?

i mean, there are just three levels: off, on-when-safe, on-even-when-unsafe?

maybe make it a enum then, with three values. maybe call them yes/no and `force`